Rakefile

```diff
@@ -168,9 +168,14 @@ def create_post_file!(filename, response, date, host)
     line.puts "components: #{components}"
     line.puts "host: #{host}"
     line.puts "status: upcoming"
+    line.puts "commit:"
     line.puts "---\n\n"
     line.puts "## Notes\n\n"
     line.puts "## Questions\n"
+    line.puts "\n"
+    line.puts "## Meeting Log\n"
```

Comment:
I think having an empty meeting log before the meeting is ugly. Can you either remove this or comment it out?

Reply:
Good idea. Done.